.github/workflows/integration_tests.yml

```diff
@@ -55,27 +56,137 @@ jobs:
           echo "refresh: $refresh_token" >> ~/.quantinuum_config
 
       - name: QIR syntax check (Quantinuum)
-        if: inputs.syntax_check || github.event_name == 'schedule'
+        if: inputs.syntax_check || github.event_name == 'schedule' || inputs.target == 'nightly'
         run: |
+          echo "### QIR syntax check (Quantinuum)" >> $GITHUB_STEP_SUMMARY
+          export CUDAQ_LOG_LEVEL="info"
+          set +e # Allow script to keep going through errors
+          test_err_sum=0
           for filename in test/NVQPP/integration/*.cpp; do
             [ -e "$filename" ] || echo "::error::Couldn't find files ($filename)"
             nvq++ -v $filename -DSYNTAX_CHECK --target quantinuum --quantinuum-machine H1-1SC
-            CUDAQ_LOG_LEVEL=info ./a.out
```

Comment:
Gettin the verbose information is useful for debugging. We could output to a file and in case of an error, output the file contents into the summary or terminal.

Reply:
The verbose information will still be there. I just put it in line 62 (new line number).

Reply:
Oh I missed that! Sorry (: